code/game/objects/items/stacks/tiles/tile_types.dm

```diff
@@ -201,6 +210,9 @@
 /obj/item/stack/tile/carpet/royalblue/fifty
 	amount = 50
 
+/obj/item/stack/tile/carpet/eighties
```

Comment:
different typepath to your other stack which is `/obj/item/stack/tile/eighties`

Reply:
fixed it, thanks for pointing out.